tests/functests/instancetype_test.go

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Common instance types func tests", func() {
+
+	Context("Cluster interaction", func() {
+
+		It("Test if client works", func() {
```

Comment:
Not sure about this test, not required I'd say.

Reply:
Yep, I will drop it later